.github/workflows/run_pathways_tests_internal.yml

```diff
@@ -0,0 +1,50 @@
+# Copyright 2024 Google LLC
```

Comment:
Could you update the year to 2025 here and below (for new files)?

Reply:
Ohh yes, good catch! Thank you!